### PR TITLE
Week 3 Lab bug fixes

### PR DESCRIPTION
**Pull request by UTpH**
Bug fixes.

### CODE REVIEW

**Comment by UTpH** [comment]
Fixed all src issues. Replaced all images with <br>, format.

**Comment by UTpH** [comment]
Resolved issues!

File: docs/chapters/03-3.md [hunk 1]
```diff
@@ -26,22 +26,23 @@ If our data exhibits stationarity, locality, and compositionality, we can exploi
...
 ### Stationarity $\Rightarrow$ parameters sharing
 
 If our data exhibits stationarity, we could use a small set of parameters multiple times across the network architecture. For example in our sparse network, Fig.3(a), we can use a set of 3 shared parameters (yellow, orange and red). The number of parameters will then drop from 9 to 3! The new architecture might even work better because we have more data for training those specific weights.
 The weights after applying sparsity and parameter sharing is called a convolution kernel.
 
-|<img src="https://i.imgur.com/E2w4QoY.png" width="300"/> | <img src="https://i.imgur.com/l08MOj9.png" width="300"/>|
-|*Figure 3(a): Before Applying Parameter Sharing* | *Figure 3(b): After Applying Parameter Sharing*|
+<img src="Figure 3(a) Before Applying Parameter Sharing.png" width="300"/> <img src="Figure 3(b) After Applying Parameter Sharing.png" width="300"/> <br>
+ *Figure 3(a): Before Applying Parameter Sharing* &nbsp; &nbsp; &nbsp; &nbsp; *Figure 3(b): After Applying Parameter Sharing*
+
```
Comment (Atcold) [inline docs/chapters/03-3.md]: I've added a table here, in order to display the figures side by side.
Please, don't remove what I've done.

File: docs/chapters/03-3.md [hunk 2]
```diff
@@ -54,11 +55,10 @@ Following are some advantages of using sparsity and parameter sharing:-
 
 Fig.4 shows an example of kernels on 1D data, where the kernel size is: 2(number of kernels) * 7(thickness of the previous layer) * 3(number of unique connections/weights).
 
-<center>![](https://i.imgur.com/IhUSnN2.png)<br>
-*Figure 4: Kernals on 1D Data*</center>
```
Comment (Atcold) [inline docs/chapters/03-3.md]: Don't remove the centering.

File: docs/chapters/03-3.md [hunk 3]
```diff
@@ -81,32 +81,34 @@ Parts of a signal can get lost if too many layers have been stacked so, addition
...
+*Figure 5: Information Representations Moving up the Hierachy*
 
-### Pooling
+## [Pooling and Covnet - Jupyter Notebook](0:40:09-0:57:36)
+### Pooling 
+<img src="Figure 6 Illustration of Pooling.png" width="350px" /><br>
+*Figure 6: Illustration of Pooling*
 
-<center>![](https://i.imgur.com/X9wUEBH.png)<br>
-*Figure 6: Illustration of Pooling*</center>
```
Comment (Atcold) [inline docs/chapters/03-3.md]: Don't remove double empty lines before a title and figure centering.

File: docs/chapters/03-3.md [hunk 4]
```diff
@@ -81,32 +81,34 @@ Parts of a signal can get lost if too many layers have been stacked so, addition
...
 
 A specific operator, $L_p$-norm, is applied to different regions (refer to Fig.6). Such an operator gives only one value per region(1 value for 4 pixels in our example). We then iterate over the whole data region-by-region, taking steps based on the stride. If we start with $m * n$ data with $c$ channels, we will end up with $\frac{m}{2} * \frac{n}{2}$ data still with $c$ channels (refer to Fig.7).
 Pooling is not parametrized; nevertheless, we can choose different polling types like max pooling, average pooling and so on. The main purpose of pooling reduces the amount of data so that we can compute in a reasonable amount of time.
 
-<center>![](https://i.imgur.com/LZUKPSo.png)<br>
-*Figure 7: Pooling results*</center>
```
Comment (Atcold) [inline docs/chapters/03-3.md]: Centering has been removed here as well.

**Comment by UTpH** [comment]
> Can you please reintroduce the `<center> ... </center>` I've added and you've removed?

This is weird, unsure how that blew off. Fixed it now!

File: docs/chapters/03-3.md [hunk 5]
```diff
@@ -121,7 +125,8 @@ First, we train both the networks on the normalized MNIST data. The accuracy of
 
 Next, we perform a random permutation of all the pixels in all the images of our MNIST dataset. This transforms our Fig.8 to Fig.9. We then train both the networks on this modified dataset.
 
+
```
Comment (Atcold) [inline docs/chapters/03-3.md]: I think `<center>` is missing here? Does it actually render well?

File: docs/chapters/03-3.md [hunk 6]
```diff
@@ -107,8 +112,7 @@ In this notebook, we train a multilayer perceptron (FC network) and a convolutio
 
 
 <img src="Figure 8 Instances from the Original MNIST Dataset.png" width="350px" /><br>
```
Comment (Atcold) [inline docs/chapters/03-3.md]: `<center>` is missing.

File: docs/chapters/03-3.md [hunk 7]
```diff
@@ -83,20 +86,22 @@ In Fig.5, while the input image contains mostly spatial information across two d
...
+<img src="Figure 6 Illustration of Pooling.png" width="350px" /><br>
+*Figure 6: Illustration of Pooling*</center>
+
 A specific operator, $L_p$-norm, is applied to different regions (refer to Fig.6). Such an operator gives only one value per region(1 value for 4 pixels in our example). We then iterate over the whole data region-by-region, taking steps based on the stride. If we start with $m * n$ data with $c$ channels, we will end up with $\frac{m}{2} * \frac{n}{2}$ data still with $c$ channels (refer to Fig.7).
 Pooling is not parametrized; nevertheless, we can choose different polling types like max pooling, average pooling and so on. The main purpose of pooling reduces the amount of data so that we can compute in a reasonable amount of time.
 
-<img src="Figure 7 Pooling results.png" width="350px" /><br>
-*Figure 7: Pooling results*
 
+<img src="Figure 7 Pooling results.png" width="350px" /><br>
```
Comment (Atcold) [inline docs/chapters/03-3.md]: There is no `<center>`.

**Comment by Atcold** [comment]
Thanks you!

File: docs/chapters/03-3.md (hunk 6 above)
Comment (UTpH) [reply docs/chapters/03-3.md]: Fig. 8 is centered. I see it in my branch

**Comment by UTpH** [comment]
Great! Sorry for the initial confusion!

**Comment by Atcold** [comment]
Haha, no worries. I should write some instructions, actually.
So, it's my bad.

**Pull request merged**